dynamo/tools/utils.py

```diff
@@ -991,28 +991,31 @@ def inverse_norm(adata: AnnData, layer_x: Union[np.ndarray, sp.csr_matrix]) -> n
         The inverse normalized data.
     """
 
+    if layer_x == "X":
```

Comment:
We may need layer X inverse_norm, we can use something like `norm_method_key = "X_norm_method" if layer is None or layer == "X" else "layers_norm_method"`.

Reply:
@Xiaojieqiu 
Can you please confirm that we still need to do inverse_norm for adata.X ?